src/foundry/client-esm/applications/sheets/ambient-light-config.d.mts

```diff
@@ -15,4 +17,56 @@ export default class AmbientLightConfig<
   RenderContext,
   Configuration,
   RenderOptions
-> {}
+> {
+  static override DEFAULT_OPTIONS: DeepPartial<DocumentSheetV2.Configuration<AmbientLightDocument.ConfiguredInstance>>;
+
+  static override PARTS: Record<string, HandlebarsApplicationMixin.HandlebarsTemplatePart>;
+
+  /**
+   * Maintain a copy of the original to show a real-time preview of changes.
+   */
+  preview: AmbientLightDocument.ConfiguredInstance | undefined;
+
+  override tabGroups: {
+    sheet: string;
+  }
+
+  protected override _preRender(context: DeepPartial<RenderContext>, options: DeepPartial<RenderOptions>): Promise<void>;
+
+  protected override _onRender(context: DeepPartial<RenderContext>, options: DeepPartial<RenderOptions>): Promise<void>;
+
+  protected override _onClose(options: DeepPartial<RenderOptions>): void;
+
+  protected override _prepareContext(options: DeepPartial<RenderOptions>): Promise<RenderContext>;
+
+  override changeTab(tab: string, group: string, options?: InexactPartial<ApplicationV2.ChangeTabOptions>): void;
```

Comment:
Is `InexactPartial` accurate here? Also I think it'd be worth applying to the interface directly instead of at the usage site.

Reply:
Justification is in the base function definition `InexactPartial is used over NullishProps because event/navElement are not called with null as a possible value, and null interferes with the defaults of force/updatePosition`

The actual context is that `force` and `updatePosition` are checked as truthy, but `event` and `navElement` should either be passing an actual event/DOM element or not.

Reply:
Alright, go ahead and move it to the interface definition and we're good then.